deployment/kubernetes/aerie-file-store-persistentvolumeclaim.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  namespace: aerie-dev
+  name: aerie-file-store
+spec:
+  accessModes:
+    - ReadWriteOnce
```

Comment:
Why is `ReadWriteOnce` used here vs `ReadWriteMany` or `ReadWriteOncePod`?

Reply:
Good find. This should be `ReadWriteMany` to allow for multi-node deployments. `ReadWriteOnce` allows one node to mount this volume as R/W, and since local development and the dev deployment only use a single node cluster, this fell through my testing.

Reply:
Change has been made!